### PR TITLE
SPE_IFA 1.1 Template Updates

### PR DESCRIPTION
**Pull request by Tiny-DM**
## What type of PR is this.
1. [ ] Bug
2. [X] Change
3. [X] Enhancement

### What have you changed and why?
Information:
Added 1.1 content to the SPE_IFA templates. The SPE templates themselves have not been updated, support for SPEX will have to be made at some point.

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [X] Yes (Please provide further detail below.)

********************************************************
Notes:
Templates are still rough around the edges and could use some more work in terms of balancing vehicle prices, for both enemies and rebels. Transferring to @ante185 for polishing.

### CODE REVIEW

File: A3A/addons/logistics/Nodes/SPE.hpp [hunk 1]
```diff
@@ -70,4 +70,72 @@ class SPE_US_M3_Halftrack : TRIPLES(ADDON,Nodes,Base)
...
     };
 };
-class SPE_FR_M3_Halftrack : SPE_US_M3_Halftrack {};
\ No newline at end of file
+class SPE_FR_M3_Halftrack : SPE_US_M3_Halftrack {};
+
+class SPE_CCKW_353 : TRIPLES(ADDON,Nodes,Base)
+{
+    canLoadWeapon = 0;
+        class Nodes
```
Comment (rautamiekka) [inline A3A/addons/logistics/Nodes/SPE.hpp:78]:
```suggestion
    class Nodes
```

File: A3A/addons/logistics/Nodes/SPE.hpp [hunk 2]
```diff
@@ -70,4 +70,72 @@ class SPE_US_M3_Halftrack : TRIPLES(ADDON,Nodes,Base)
...
+        {
+            offset[] = {0,-2.1,-0.7};
+            seats[] = {0,1};
+        };
+    };
+};
+class SPE_CCKW_353_M2 : SPE_CCKW_353 {};
+class SPE_CCKW_353_Open : TRIPLES(ADDON,Nodes,Base)
+{
+        class Nodes
```
Comment (rautamiekka) [inline A3A/addons/logistics/Nodes/SPE.hpp:105]:
```suggestion
    class Nodes
```

File: A3A/addons/logistics/Nodes/SPE.hpp [hunk 3]
```diff
@@ -70,4 +70,72 @@ class SPE_US_M3_Halftrack : TRIPLES(ADDON,Nodes,Base)
...
+        {
+            offset[] = {0,-2.1,-0.7};
+            seats[] = {0,1};
+        };
+    };
+};
+
+class SPE_US_G503_MB : TRIPLES(ADDON,Nodes,Base)
+{
+        class Nodes
```
Comment (rautamiekka) [inline A3A/addons/logistics/Nodes/SPE.hpp:132]:
```suggestion
    class Nodes
```

**Review by rautamiekka — CHANGES_REQUESTED** [review]

**Comment by ante185** [comment]
- Set the US militia up with their thematically appropriate weapons.
- Set the WEH militia and police up themed after the forces of Vichy France.
- Made WEH military and SF not share uniforms. WEH snipers are now slightly camouflaged. 
- Shuffled vehicles around until i couldn't stand my indecision anymore.
- Updated the civ template so that it no longer spawns military trucks or military utility vehicles.
- Gave rebels their appropriate vests and backpacks.
- Set the factions with appropriate names and radio protocols.
- Gave US and WEH a distinct looking police force.
- Several passed on faction small arms to make sure they were appropiate for the faction and the military/militia/police units that will use them.

File: A3A/addons/core/Templates/Templates/SPE_IFA/SPE_IFA_AI_US.sqf [hunk 4]
```diff
@@ -20,19 +20,21 @@
...
+["vehiclesLightUnarmed", ["SPE_US_G503_MB_Armoured"]] call _fnc_saveToTemplate;
+["vehiclesLightArmed", ["SPE_US_M16_Halftrack", "SPE_M20_AUC","SPE_M20_AUC","SPE_FR_M20_AUC","SPE_M8_LAC","SPE_M8_LAC_ringMount","SPE_US_G503_MB_M1919_Armoured","SPE_US_G503_MB_M2_Armoured","SPE_US_G503_MB_M2","SPE_US_G503_MB_M2_PATROL","SPE_US_G503_MB_M1919_PATROL"]] call _fnc_saveToTemplate;
+["vehiclesTrucks", ["SPE_CCKW_353","SPE_CCKW_353_Open","SPE_CCKW_353_M2","SPE_US_M3_Halftrack_Unarmed", "SPE_US_M3_Halftrack_Unarmed_Open"]] call _fnc_saveToTemplate;
+["vehiclesCargoTrucks", ["SPE_CCKW_353_Open"]] call _fnc_saveToTemplate;
+["vehiclesAmmoTrucks", ["SPE_US_M3_Halftrack_Ammo","SPE_CCKW_353_Ammo","SPE_CCKW_353_Ammo"]] call _fnc_saveToTemplate;
+["vehiclesRepairTrucks", ["SPE_US_M3_Halftrack_Repair","SPE_CCKW_353_Repair","SPE_CCKW_353_Repair"]] call _fnc_saveToTemplate;
+["vehiclesFuelTrucks", ["SPE_US_M3_Halftrack_Fuel","SPE_CCKW_353_Fuel","SPE_CCKW_353_Fuel"]] call _fnc_saveToTemplate;
+["vehiclesMedical", ["SPE_US_M3_Halftrack_Ambulance","SPE_CCKW_353_Ambulance","SPE_US_G503_MB_Ambulance"]] call _fnc_saveToTemplate;
+["vehiclesLightAPCs", []] call _fnc_saveToTemplate;
```
Comment (jaj22) [inline A3A/addons/core/Templates/Templates/SPE_IFA/SPE_IFA_AI_US.sqf:32]: Remember that if you want to expand truck usage, one option is to add them to lightAPCs and/or APCs and force the price down in vehicle attributes. I think I did that in some of the Vietnam NVA templates.

Not sure if that's necessary here though. Depends how plausible it is to have large quantities of the APCs.

Comment (ante185) [reply A3A/addons/core/Templates/Templates/SPE_IFA/SPE_IFA_AI_US.sqf:32]: this should be fine for this faction

**Review by Bob-Murphy — APPROVED** [review]
lgtm

**Pull request merged**